internal/interpreter/highlight.rs

```diff
@@ -116,7 +241,7 @@ fn repeater_path(elem: &ElementRc) -> Option<Vec<String>> {
     let enclosing = elem.borrow().enclosing_component.upgrade().unwrap();
     if let Some(parent) = enclosing.parent_element.upgrade() {
         if parent.borrow().repeated.is_none() {
-            // This is not a repeater, it is possibily a popup menu which is not supported ATM
+            // This is not a repeater, it is possibility a popup menu which is not supported ATM
```

Comment:
That's not the right spelling fix.

```suggestion
            // This is not a repeater, it is possibly a popup menu which is not supported ATM
```

(or maybe?)